src/node.cc

```diff
@@ -111,7 +111,7 @@ typedef int mode_t;
 #include <unistd.h>  // setuid, getuid
 #endif
 
-#if defined(__POSIX__) && !defined(__ANDROID__)
+#if defined(__POSIX__) && !defined(__ANDROID__) && !defined(__CloudABI__)
```

Comment:
IMHO since this is used in three places, it should be aliased to something like `UNSANDBOXED_POSIX` or `defined(__POSIX__) && !SANDBOXED_POSIX`

Reply:
I've just added a definition, `HAVE_POSIX_CREDENTIALS`, that is declared when building for systems that support `<pwd.h>`, `<grp.h>`, `set*uid()`, etc. Does that look all right?

Reply:
I yield to @cjihrig.
Maybe revisit in the future if there are more usages.

Reply:
All right. I've just rolled back the `HAVE_POSIX_CREDENTIALS` changes.